tests/app/conftest.py

```diff
@@ -847,10 +847,10 @@ def email_2fa_code_template(notify_db, notify_db_session):
         content=(
             'Hi ((name)),'
             ''
-            'To sign in to GOV.​UK Notify please open this link:'
+            'To sign in to Notify please open this link:'
             '((url))'
```

Comment:
Should this test / template also change?

Reply:
I don't think it needs to totally match the live one, but yes I'll modify it so it's clear it's using a code and not a link (to avoid confusion)